pkg/odo/cli/feature/features.go

```diff
@@ -16,50 +13,25 @@ type OdoFeature struct {
 	// isExperimental indicates whether this feature should be considered in early or intermediate stages of development.
 	// Features that are not experimental by default will always be enabled, regardless of the experimental mode.
 	isExperimental bool
-
-	// description provides a human-readable overview of this feature.
-	// Note that this will be visible by the end users if this feature is experimental and the experimental mode is enabled.
-	description string
 }
 
 var (
 	// GenericRunOnFlag is the feature supporting the `--run-on` generic CLI flag.
 	GenericRunOnFlag = OdoFeature{
 		id:             "generic-run-on",
```

Comment:
Is this id really needed ? It seems not used in the code

Reply:
Indeed, it no longer seems necessary. Removing it..